odoo-openmrs/src/main/java/com/ozonehis/eip/odoo/openmrs/client/OdooUtils.java

```diff
@@ -12,11 +12,23 @@
 import java.lang.reflect.Field;
 import java.util.HashMap;
 import java.util.Map;
+import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
 
 @Slf4j
+@Component
 public class OdooUtils {
 
+    @Getter
+    private static String odooCustomerWeightField;
+
+    @Value("${odoo.customer.weight.field}")
+    public void setOdooCustomerWeightField(String value) {
+        odooCustomerWeightField = value;
+    }
+
```

Comment:
Based on this implementation, I’d assume that creating a custom deserializer/serializer that can be reused across all custom field configurations isn't straightforward. Otherwise trying to do so this way could pose challenges in terms of maintenance and scalability, especially when adding more custom fields.

Reply:
Yeah. So the issue is there is no actual JSON involved but it is being used for converting an object to a map, that's why we have to do this implementation. Lets discuss in Ozone Biweekly call